### PR TITLE
Update CAN lab to PlatformIO and F042K6

### PR DESCRIPTION
**Pull request by dragonlock2**
- added back solution code
- updated lab2.md to match

### CODE REVIEW

File: platformio.ini [hunk 1]
```diff
@@ -70,3 +70,23 @@ src_filter = +<src/*> -<src/main.cpp> +<solutions/lab1.5.cpp>
...
+extends = base
+src_filter = +<src/*> -<src/main.cpp> +<solutions/lab2.3.cpp>
+
+[env:solution_2_4]
+extends = base
+src_filter = +<src/*> -<src/main.cpp> +<solutions/lab2.4.cpp>
+
+[env:solution_2_5]
+extends = base
+src_filter = +<src/*> -<src/main.cpp> +<solutions/lab2.5.cpp>
```
Comment (ducky64) [inline platformio.ini:92]: style nit: newlines at end of files (and for all files)

File: lab2.md [hunk 2]
```diff
@@ -86,14 +93,13 @@ Start with this code skeleton in your `src/main.cpp`:
...
+DigitalOut led1(LED1);
 
-DigitalIn btn(P0_4);
+RawSerial serial(SERIAL_TX, SERIAL_RX, 115200);
 
-RawSerial serial(P0_8, NC, 115200);
+DigitalIn btn(D8, PullUp);
 
-CAN can(P0_28, P0_29);
+CAN can(D10, D2);
```
Comment (ducky64) [inline lab2.md:102]: One thing that would be nice is to have the pin name after as comment, eg `// RX, TX`
(and similarly for all the code examples)

**Review by ducky64 — APPROVED** [review]
Two style issues, but otherwise good.

I'll try to get graphics later tonight,

**Comment by dragonlock2** [comment]
diagrams to be added in future PR

**Pull request merged**